client/lbclient/datasources.local.js

```diff
@@ -2,6 +2,6 @@ var GLOBAL_CONFIG = require('../../global-config');
 
 module.exports = {
   remote: {
-    url: GLOBAL_CONFIG.restApiUrl
+    url: 'http://localhost:3000' + GLOBAL_CONFIG.restApiUrl
```

Comment:
Note that host + port value must be synchronized with settings in `server/config.json`. When you change port in `server/config.json`, the client app should automatically pick it up after rebuild via grunt.